container/ip-address-expansion/presentation/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
Seemed like two copies of same file exists in this pull request

Reply:
yes, one is in the "GUIsrc" the other is the build output (We probably shouldn't check in the build output)